examples/web-getting-deployed/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+
+import org.jetbrains.kotlin.gradle.targets.js.webpack.KotlinWebpack
+
+val ktorVersion = "1.5.4"
+
+plugins {
+    kotlin("multiplatform") version "1.4.32"
+    application
+    war
+    id("org.jetbrains.compose") version "0.0.0-web-dev-12"
+    id("org.gretty") version "3.0.4"
+}
+
+repositories {
+    mavenCentral()
+    maven(url = "https://maven.pkg.jetbrains.space/public/p/compose/dev")
+    maven(url = "https://plugins.gradle.org/m2/")
+}
+
+kotlin {
+    jvm()
+    js(IR) {
+        browser()
+        binaries.executable()
+    }
+    sourceSets {
+        val jsMain by getting {
+            dependencies {
+                implementation(compose.web.web)
+                implementation(compose.runtime)
+            }
+        }
+        val jvmMain by getting {
+            dependencies {
+                implementation(compose.runtime) // Can this be decoupled?
```

Comment:
Is compose.runtime needed in jvmMain for this example? Doesn't it work without it?

Reply:
Thanks for the question: This was indeed necessary to compile, but a side-effect of having clumsily modeled the Client and Server as 'targets' of the same project - where the Compose Compiler apparently expects to find runtime support in all targets of any project it is applied to.  As noted below; this modeling is not clean and should not be held as an example.  Part of my reason for voluntarily declining the PR.